packages/ckeditor5-package-generator/tests/utils/copy-files.js

```diff
@@ -158,6 +158,27 @@ describe( 'lib/utils/copy-files', () => {
 		].join( '\n' ) );
 	} );
 
+	it( 'creates the ".gitignore" file for typescript', () => {
```

Comment:
```suggestion
	it( 'creates the ".gitignore" file for TypeScript', () => {
```

The same as with `javascript`.